src/www/services_dhcpv6.php

```diff
@@ -34,24 +34,120 @@
+    </tr>
+  EOD;
+
+
+  echo <<<EOD
+    <section class="page-content-main">
+      <div class="container-fluid">
+        <div class="row">
+          <section class="col-xs-12">
+            <div class="tab-content content-box col-xs-12">
+              <form method="post" name="iform" id="iform">
+                <div class="table-responsive">
+                  <table class="table table-striped opnsense_standard_table_form">
+                    <tr>
+                      <td style="width:22%"></td>
+                      <td style="width:78%; text-align:right">
+                        <div style='height: 15px;'></div>
+                      </td>
+                    </tr>
+                    <tr>
+                      <td><i class="fa fa-info-circle text-muted"></i> $enable_label</td>
+                      <td>
+                        <input name="enable" type="checkbox" value="yes" $enable_input/>
+                        <strong>$enable_descr</strong>
+                      </td>
+                      $range_tr
+                    </tr>
+                    <tr>
+                      <td>&nbsp;</td>
+                      <td>
+                        <input name="if" type="hidden" value="$if" />
+                        <input name="submit" type="submit" class="formbtn btn btn-primary" value="$save_btn_text"/>
+                      </td>
+                    </tr>
+                  </table>
+                </div>
+              </form>
+            </div>
+          </section>
+        </div>
+      </div>
+    </section>
+  EOD;
+  include("foot.inc");
+}
+
+function process_track6_form($if)
+{
+    $dhcpdv6cfg = &config_read_array('dhcpdv6');
+    $this_server = [];
+    if (isset($dhcpdv6cfg[$if]) && isset($dhcpdv6cfg[$if]['ramode'])) {
+        /* keep ramode for router advertisements so we can use this field to disable the service when in tracking mode */
+        $this_server['ramode'] = $dhcpdv6cfg[$if]['ramode'];
+    }
+    if (empty($_POST['enable'])) {
+        $this_server['enable'] = '-1';
+    }
+    $dhcpdv6cfg[$if] = $this_server;
+    write_config();
+    reconfigure_dhcpd();
+    filter_configure();
+    header(url_safe('Location: /services_dhcpv6.php?if=%s', array($if)));
+}
+
+
 function reconfigure_dhcpd()
 {
     system_resolver_configure();
     dhcpd_dhcp6_configure();
     clear_subsystem_dirty('staticmapsv6');
 }
 
+$if = null;
+$act = null;
 if ($_SERVER['REQUEST_METHOD'] === 'GET') {
     // handle identifiers and action
     if (!empty($_GET['if']) && !empty($config['interfaces'][$_GET['if']]) &&
         isset($config['interfaces'][$_GET['if']]['enable']) &&
         (is_ipaddr($config['interfaces'][$_GET['if']]['ipaddrv6']) ||
-        !empty($config['interfaces'][$_GET['if']]['dhcpd6track6allowoverride']))) {
+        !empty($config['interfaces'][$_GET['if']]['track6-interface']))) {
         $if = $_GET['if'];
-    } else {
-        /* if no interface is provided this invoke is invalid */
```

Comment:
wouldn't remove this at first glance, the page shows garbage when invoked this way?